client/src/components/Settings.js

```diff
@@ -1,40 +1,65 @@
-import React from "react";
+import React, { useState } from "react";
 import NavBar from "./NavBar";
 import Form from "react-bootstrap/Form";
+import axios from "axios";
+import { constants, errors, routes } from "../Common";
+import { ToastsStore } from "react-toasts";
 
-class Account extends React.Component {
-	constructor(props) {
-		super(props);
-	}
+export default function Settings() {
+	const token = localStorage.getItem(constants.TOKEN);
+	const [curPwd, setCurPwd] = useState("");
+	const [newPwd, setNewPwd] = useState("");
 
-	render() {
-		return <div className="account__container">
-			<NavBar />
-			<div className={"account__header"}>
-				<h1> Account Management </h1>
-			</div>
-			<div>
-				Update Password
-				<Form.Group className="mb-3" controlId="formBasicEmail">
-					<Form.Control type="email" placeholder="Enter Current Password" />
-				</Form.Group>
-				<Form.Group className="mb-3" controlId="formBasicEmail">
-					<Form.Control type="email" placeholder="Enter New Password" />
-				</Form.Group>
-				<button type="button" className="btn btn-primary btn-lg">Update</button>
-			</div>
-			<div>
-				Delete Account
-				<Form.Group className="mb-3" controlId="formBasicEmail">
-					<Form.Control type="email" placeholder="Enter Password" />
-				</Form.Group>
-				<Form.Group className="mb-3" controlId="formBasicEmail">
-					<Form.Control type="email" placeholder="Confirm Password" />
-				</Form.Group>
-				<button type="button" className="btn btn-primary btn-lg">Submit</button>
-			</div>
-		</div>;
-	}
-}
+	const updatePassword = () => {
+		if (curPwd === constants.EMPTY) {
+			ToastsStore.error(errors.CURPWD);
+			return;
+		} else if (newPwd === constants.EMPTY) {
+			ToastsStore.error(errors.NEWPWD);
+			return;
+		}
 
-export default Account;
\ No newline at end of file
+		const data = {
+			email: localStorage.getItem(constants.EMAIL),
+			currentPassword: curPwd,
+			newPassword: newPwd,
+			name: localStorage.getItem(constants.NAME)
+		};
+		axios.post(routes.UPDATE_PWD, data, { headers: { "Authorization": `Bearer ${token}` } })
```

Comment:
As mentioned before, you can fetch the token just before this line

Reply:
Which line exactly? Before line 28?